src/js/constants/DefaultErrorMessages.js

```diff
@@ -24,8 +26,8 @@ const DefaultErrorMessages = [
   //
   // NOTE: Every time you are about to write a description for the error
   //       message make sure that it sounds natural *both* on it's own
-  //       (ex. 'Must be one of {{value}}') AND when prefixed with the
-  //       expression 'The service' (ex. 'The service must be one of {{value}}')
+  //       (ex. 'Must be one of ||value||') AND when prefixed with the
```

Comment:
so `i18nMark` will replace `||` with `{{` ?

Reply:
We are replacing these values with variables manually in `ErrorMessageUtil`(using regex to match the `{{ }}` pattern). Lingui doesn't like this syntax in its message extraction/translation catalog so I changed it to `||` instead to get around the issue